$NetBSD: patch-libgcc_config_t-darwin-min-11,v 1.2 2025/06/08 07:37:46 wiz Exp $

libgcc, Darwin: From macOS 11, make that the earliest supported.

For libgcc, we have (so far) supported building a DSO that supports
earlier versions of the OS than the target.  From macOS 11, there are
APIs that do not exist on earlier OS versions, so limit the libgcc
range to macOS11..current.

See https://github.com/gcc-mirror/gcc/blob/master/libgcc/config/t-darwin-min-11.

--- libgcc/config/t-darwin-min-11.orig	2025-05-23 11:02:08.248259924 +0000
+++ libgcc/config/t-darwin-min-11
@@ -1,3 +1,6 @@
 # Support building with -mmacosx-version-min back to macOS 11.
 DARWIN_MIN_LIB_VERSION = -mmacosx-version-min=11
 DARWIN_MIN_CRT_VERSION = -mmacosx-version-min=11
+# Support building with -mmacosx-version-min back to macOS 11.
+DARWIN_MIN_LIB_VERSION = -mmacosx-version-min=11
+DARWIN_MIN_CRT_VERSION = -mmacosx-version-min=11
